.github/labels.yml

```diff
@@ -5,219 +5,36 @@
 - name: "good first issue"
   color: 7057ff
   description: "Good for newcomers"
-- name: "help wanted"
-  color: 008672
-  description: "Extra attention is needed"
 
 ###
 ### Areas
 #
-- name: area/ux
+- name: ux
   color: 00A4E0
   description: "Area: UX"
-- name: area/node
-  color: 0bb1ed
-  description: "Area: Drand Node"
-- name: area/relay
-  color: 0EB4F0
-  description: "Area: Drand Relay"
-- name: area/client
-  color: 10B6F2
-  description: "Area: Drand Client"
-- name: area/docs
+- name: docs
   color: 13B9F5
   description: "Area: Documentation"
-- name: area/testing
+- name: testing
```

Comment:
We can assign issues to a milestone without needing a tag for that milestone, I planned to assign the "testing" label to the flaky tests, but not to the "logging" issues that aren't related to tests, these might go under "ux", since I'd argue that good logging helps with the operator UX.

But we might want to add a label "refactoring" or "cleanup", now that I think about it since we want to refactor some portions but also want new features and stuff.